trust-store-trust-policy-specification.md

```diff
@@ -111,15 +111,14 @@ Trust policy for the scenario where ACME Rockets uses artifacts signed by themse
             // Policy for with custom verification policy
             "name": "use-expired-image",
             "registryScopes": [ "registry.acme-rockets.io/software/legacy/metrics" ],
-            "signatureVerification":
-            {
+            "signatureVerification": {
               "level" : "strict",
-              "override" :
-              {
-                "expiry" : "skip"
+              "override" : {
+                "expiry" : "log",
+                "revocation" : "skip"
               }
             },
-            "trustStore": "ca:acme-rockets",
+            "trustStores": ["ca:acme-rockets"],
```

Comment:
Should we provide an example of two `["ca:acme-rockets-staging", "ca:acme-rockets-prod"]`